server/api.ts

```diff
@@ -1059,6 +1059,20 @@ router.getAsync("/stage-stats", async (req, res) => {
   res.send(stageStats);
 });
 
+/**
+ * POST /api/stage-stats
+ * Edit stats for a tournament stage
+ * Params:
+ *   - stats: updated StageStats object
+ */
+router.postAsync("/stage-stats", ensure.isAdmin, async (req, res) => {
+  const updatedStageStats = await StageStats.findOneAndUpdate(
+    { tourney: req.body.stats.tourney, stage: req.body.stats.stage },
+    req.body.stats
+  );
```

Comment:
in order to return the updated document, i think you need to pass a third argument `{new: true}` here

Reply:
done